lib/index.js

```diff
@@ -33,6 +49,11 @@ export default {
         const filePath = TextEditor.getPath();
         const fileText = TextEditor.getText();
         const ignoreFile = helpers.find(filePath, '.eslintignore');
+        const eslintrc = helpers.find(filePath, ['.eslintrc', '.eslintrc.js', '.eslintrc.json', '.eslintrc.yml']);
+
+        if (this.disableWhenNoEslintrcFileInPath && !eslintrc) {
```

Comment:
I would simply use `atom.config.get('linter-eslint. disableWhenNoEslintrcFileInPath')` instead of registering an event handler